dca632fffe7697e0.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "dca632fffe7697e0",
```

Comment:
This EUI and also the file name (not extension) should be all uppercase, please update and then we can merge.

Reply:
OK